sequence_processing_pipeline/Pipeline.py

```diff
@@ -158,6 +160,49 @@ def add(self, job):
         else:
             raise PipelineError("object is not a Job object.")
 
+    def validate(self, sample_sheet_path):
+        '''
+        Performs additional validation for sample-sheet on top of metapool.
+        :param sample_sheet_path: Path to sample-sheet.
+        :return: If successful, an empty list of strings and a valid
+                 sample-sheet. If unsuccessful, a list of warning and error
+                 messages and None.
+        '''
+        # validate the sample-sheet using metapool package.
+        sheet = KLSampleSheet(sample_sheet_path)
+        msgs, val_sheet = quiet_validate_and_scrub_sample_sheet(sheet)
+        if val_sheet:
```

Comment:
What's the return value of `val_sheet` from `quiet_validate_and_scrub_sample_sheet`? If `None`, then (1) this should be `if val_sheet is None` and we don't need the two `return` statement; only one outside the if.

Reply:
It's (None, sample-sheet-obj) if it's a valid sample-sheet and ([error messages], None) if not. Good point, thanks! I'll make the change.

Reply:
It's (None, sample-sheet-obj) if it's a valid sample-sheet and ([error messages], None) if not. Good point, thanks! I'll make the change.